ci/build_alpine_container/README.md

```diff
@@ -0,0 +1,17 @@
+This directory contains a Dockerfile that builds an envoy image
```

Comment:
As long as this image uses glibc, we could replace the ubuntu one with this if it would make people happier. I don't really care.